src/account-info/AccountDetailsScreen.js

```diff
@@ -71,6 +95,12 @@ class AccountDetailsScreen extends PureComponent<Props> {
 }
 
 export default connect<SelectorProps, _, _>((state, props) => ({
-  user: getAccountDetailsUserForEmail(state, props.navigation.state.params.email),
```

Comment:
Okay. The lates push creates some new selectors which are used here - these selectors are in their separate commits.

Reply:
Yeah, this makes sense - a migration commit _should_ be an NFC. If it's not, then it's not strictly a migration.

Reply:
Yeah, I'd say that at least for the term "refactor" -- a change that isn't NFC isn't purely a refactor, and "pure refactor" is a close synonym of "NFC".

I'd actually use "migration" somewhat more broadly: e.g. I might talk about "migrating" to a new major version of some dependency (especially a big dependency, like RN itself), and that's typically not NFC. But in this case, the kind of migration we're going for is pretty much a pure refactor.

... Oh, actually, here's another example of "migrate" -- for the commit that replaces the use of `getAccountDetailsUserForEmail` with `getUserForEmail`, I would actually describe that as *migrating* from `getAccountDetailsUserForEmail` to `getUserForEmail`. (I noticed this example as I found myself typing exactly that in the comment below.) Even though, in edge cases, it may not behave quite the same, and so it isn't NFC.

Reply:
So, I think this sequence of code changes is now good!

There's some work I have to do to get to that conclusion, in the case of the change that migrates from `getAccountDetailsUserForEmail` to `getUserForEmail`: ask the questions, "what are the cases where this can behave differently? what's the old behavior in those cases, and what's the new behavior? could the new behavior be worse?", and study the code to answer them.

So the commit message should reflect that work. :slightly_smiling_face: I'll edit it to add that.